src/main/java/stream/flarebot/flarebot/FlareBot.java

```diff
@@ -250,118 +237,29 @@ public static Prefixes getPrefixes() {
         return prefixes;
     }
 
-    private void init(String tkn) throws InterruptedException {
+    public void init() throws InterruptedException {
         LOGGER.info("Starting init!");
-        token = tkn;
         manager = new FlareBotManager();
         RestAction.DEFAULT_FAILURE = t -> {
         };
-        try {
-            clients = new JDA[WebUtils.getShards(tkn)];
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
         Runtime.getRuntime().addShutdownHook(new Thread(this::stop));
 
-        latch = new CountDownLatch(1);
         events = new Events(this);
         LOGGER.info("Starting builders");
+
         try {
-            JDABuilder builder = new JDABuilder(AccountType.BOT)
-                    .addEventListener(events)
-                    .addEventListener(new ModlogEvents())
-                    .setToken(tkn)
-                    .setHttpClientBuilder(client.newBuilder())
+            shardManager = new DefaultShardManagerBuilder()
+                    .addEventListeners(events)
+                    .setToken(config.getString("bot.token").get())
                     .setAudioSendFactory(new NativeAudioSendFactory())
-                    .setBulkDeleteSplittingEnabled(false);
-            if (clients.length == 1) {
-                clients[0] = builder.buildBlocking(JDA.Status.AWAITING_LOGIN_CONFIRMATION);
-                Thread.sleep(5000);
-            } else {
-                builder = builder.setReconnectQueue(new SessionReconnectQueue());
-                for (int i = 0; i < clients.length; i++) {
-                    clients[i] = builder.useSharding(i, clients.length).buildBlocking(JDA.Status.AWAITING_LOGIN_CONFIRMATION);
-                    Thread.sleep(5000); // 5 second backoff
-                }
-            }
+                    .setShardsTotal(-1)
```

Comment:
This isn't needed, it's the default behaviour of the builder

Reply:
Possibly, but it still makes it easier to switch to like 2 shards when we want to test multi-shard things